docs/concepts/configuration-documents/overview.md

```diff
@@ -0,0 +1,255 @@
+resources:
+  - name: example key value
+    type: Microsoft.Windows/Registry
+    properties:
+      keyPath:   HKCU\example\key
+      valueName: Example
+      valueData:
+        String: This is an example.
+```
+
+The example document defines a single resource instance named `example key value`. The instance
+uses the `Microsoft.Windows/Registry` resource to declare the following desired state:
+
+- The `example\key` registry key should exist in the system's current user hive.
+- The `example\key` registry key should have a value called `Example`.
+- The `Example` value should be the string `This is an example`.
+
+The example document is _declarative_. It describes the desired state, not how to put the system
+into that state. It relies on the `Microsoft.Windows/Registry` resource to handle getting, testing,
+and setting the state of the registry key and value.
+
+For more information about the structure and validation of configuration documents, see
+[DSC Configuration document schema reference][03].
+
+### Defining resource instances
+
+As shown in the prior example, configuration documents include a collection of resource instances.
+Together, the instances describe the desired state of a system. A configuration document can
+include any number of resource instances.
+
+A resource instance declaration always includes:
+
+- `name` - A short, human-readable name for the instance that's unique in the document. This name
+  is used for logging and it helps describe the purpose of the instance.
+- `type` - The [fully qualified type name][04] of the resource that DSC should use to manage the
+  instance.
+- `properties` - The desired state for the instance. DSC validates the values against the
+  resource's instance schema.
+
+Configuration documents can't include the same instance more than once. Declaring the same instance
+with different properties leads to enforcement cycling, where each declaration enforces an
+incompatible state for the instance on every run.
+
+## Getting the current state of a configuration
+
+The `dsc config get` command retrieves the current state of the resource instances defined in a
+configuration document. When you use this command, DSC also:
+
+- Collects any message emitted by the resources during the operation.
+- Indicates whether any of the resources raised an error.
+- Provides metadata about the operation as a whole and for each resource instance.
+
+```sh
+dsc config get --file ./example.config.dsc.yaml
+```
+
+```yaml
+metadata:
+  Microsoft.DSC:
+    version: 3.0.0
+    operation: get
+    executionType: actual
+    startDatetime: 2025-02-24T16:09:40.671454400-06:00
+    endDatetime: 2025-02-24T16:09:41.850086300-06:00
+    duration: PT1.1786319S
+    securityContext: restricted
+results:
+- metadata:
+    Microsoft.DSC:
+      duration: PT0.2751153S
+  name: example key value
+  type: Microsoft.Windows/Registry
+  result:
+    actualState:
+      keyPath: HKCU\example\key
```

Comment:
Enclose in single quotes so that the backslash doesn't need to be escaped

```suggestion
      keyPath: 'HKCU\example\key'
```

Reply:
The backslash doesn't need escaping in YAML for unquoted strings:

```console
@'
unquoted:       foo\bar\baz
double_quoted: "foo\\bar\\baz"
single_quoted: 'foo\bar\baz'
'@ | ConvertFrom-Yaml | Format-List

Name  : unquoted
Value : foo\bar\baz

Name  : double_quoted
Value : foo\bar\baz

Name  : single_quoted
Value : foo\bar\baz
```

Reply:
I guess it's just GitHub's syntax coloring then